tasking/pulp/tasking/constants.py

```diff
@@ -4,6 +4,20 @@
     # The interval in seconds for which the Celery Process monitor thread sleeps between
     # checking for missing Celery processes.
     CELERY_CHECK_INTERVAL=60,
+    # The interval in seconds during which the Celery Processes being monitored need to report a
+    # hearbeat to be considered active.
+    CELERY_TIMEOUT_SECONDS=200,
+    # The interval in seconds with which a secondary resource manager tries to acquire the lock
+    # that is needed to be the primary resource manager.
+    CELERY_TICK_DEFAULT_WAIT_TIME=90,
```

Comment:
So I looked at the code, and the comment was wrong. This value is used by the celerybeat only. It's how long celerybeat sleeps between trying to acquire the lock. I am changing this to `CELERYBEAT_LOCK_RETRY_TIME` and updating the comment.